File: chapter_001/moon.py
#!/usr/bin/env python3

import math

moon = 384_400
car = 80
train = 300
hour = 24

print(f"take {moon / car / hour} days to go around the moon by car.")
print(f"take {moon / train / hour} days to go around the moon by train.")
print(f"take {math.floor(moon / car / hour):.2f} days to go around the moon by car.")
print(f"take {math.floor(moon / train / hour):2f} days to go around the moon by train.")
print(f"take {round(moon / car / hour, 2)} days to go around the moon by car.")
print(f"take {round(moon / train / hour, 2)} days to go around the moon by train.")

File: chapter_001/moon.rs
fn round_2_decimals(x: f64) -> f64 {
    (x * 100.0).round() / 100.0
}

fn main() {
    let moon: f64 = 384400.0;
    let car: f64 = 80.0;
    let train: f64 = 300.0;
    let hour: f64 = 24.0;

    let mut result: f64;
    result = moon / car / hour;
    println!("take {}days to the moon by car.", round_2_decimals(result));

    result = moon / train / hour;
    println!("take {}days to the moon by train.", round_2_decimals(result));
}